Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -25332,4 +25359,209 @@
         <EditorCategories value="ObjectType:Structure"/>
         <HotkeyCategory value="Unit/Category/AP_ZergUnits"/>
     </CUnit>
+    <CUnit id="AP_MercZergling" parent="AP_Zergling">
+        <Name value="Unit/Name/AP_MercZergling"/>
+        <Description value="Button/Tooltip/AP_MercZergling"/>
+        <LifeStart value="120"/>
+        <LifeMax value="120"/>

```

Comment:
SC1 Devouring Ones have 3 base armor

Reply:
I don't mind setting that, but seemed like it might be too strong